control_panel_api/management/commands/migrate_lambda_data_0_dump_user_policies.py

```diff
@@ -0,0 +1,58 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+from django.core.management.base import BaseCommand
+
+from control_panel_api.models import User
+
+
+READWRITE = 'readwrite'
```

Comment:
Don't need this in this file any more

Reply:
That was an artefact of the Copy&Paste. Well spotted.